app/recipe/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+
+
+class RecipeManager(models.Manager):
+    def create_recipe(self, recipe: dict, ingredients: list):
+        recipe = Recipe.objects.create(**recipe)
+        recipe.add_ingredients(ingredients=ingredients)
+
+        return recipe
+
+
+class Recipe(models.Model):
+    objects = RecipeManager()
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    name = models.CharField(max_length=255)
```

Comment:
Best to just use `TextField` for postgres there is no difference in performance.